pkg/controller/controller_utils.go

```diff
@@ -400,6 +400,19 @@ const (
 	// SuccessfulDeletePodReason is added in an event when a pod for a replica set
 	// is successfully deleted.
 	SuccessfulDeletePodReason = "SuccessfulDelete"
+
+	// FailedCreatePodAction is added in an event and in a replica set condition
+	// when a pod for a replica set is failed to be created.
+	FailedCreatePodAction = ""
```

Comment:
this seems odd... there's no identifying info for this action, and it is identical to FailedDeletePodAction?

Reply:
Currently, I cannot decide what proper Actions should be. Also in other controllers. So, Most of the evevtf have not been update. I will update them and have a discussion in this PR to name the Actions.